src/telemTestTool.pro

```diff
@@ -2,7 +2,7 @@
     error( "Couldn't find the common.pri file!" )
 }
 
-QT += widgets
+QT += widgets serialport
```

Comment:
Stuff like this is what the common.pri file is for, did you try putting it there?